packages/twenty-front/src/modules/settings/developers/utils/format-expiration.ts

```diff
@@ -11,6 +11,9 @@ export const formatExpiration = (
 ) => {
   if (isNonEmptyString(expiresAt)) {
     const dateDiff = beautifyDateDiff(expiresAt, undefined, short);
+    if (dateDiff === '+∞') {
+      return withExpiresMention ? 'Never expires' : 'Never';
```

Comment:
I don't think we should do the condition on the beautified date, it does not seems robust? Can we do it on the expiresAt directly?

Reply:
Yes I can create a dedicated util